src/Stripe.net/Services/ExternalAccounts/StripeExternalAccountCreateOptions.cs

```diff
@@ -0,0 +1,20 @@
+using System.Collections.Generic;
+using Newtonsoft.Json;
+
+namespace Stripe
+{
+    public class StripeExternalAccountCreateOptions : StripeBaseOptions
+    {
+        [JsonProperty("external_account")]
+        public string ExternalAccountTokenId { get; set; }
+
+        [JsonProperty("external_account")]
+        public StripeAccountBankAccountOptions ExternalAccountBankAccount { get; set; }
+
```

Comment:
We don't support card on creation because it's blocked in the API by default. would be trivial to add if needed since there's `StripeAccountCardOptions`